### PR TITLE
webboot: add Ubuntu and Arch Linux

### PR DESCRIPTION
**Pull request by orangecms**
this also reenables the original `tinycore` command

### CODE REVIEW

File: webboot/webboot.go [hunk 1]
```diff
@@ -45,10 +50,49 @@ var (
 	wifi    = flag.String("wifi", "GoogleGuest", "[essid [WPA [password]]]")
 	bookmark = map[string]*webboot.Distro{
 		// TODO: Fix webboot to process the tinycore's kernel and initrd to boot from instead of using our customized kernel
-		// "tinycore": &webboot.Distro{"boot/vmlinuz64", "/boot/corepure64.gz", "console=tty0", "http://tinycorelinux.net/10.x/x86_64/release/TinyCorePure64-10.1.iso"},
-		"Tinycore": &webboot.Distro{"/bzImage", "/boot/corepure64.gz", "memmap=4G!4G console=tty1 root=/dev/pmem0 loglevel=3 cde waitusb=5 vga=791", "http://tinycorelinux.net/10.x/x86_64/release/TinyCorePure64-10.1.iso"},
+		"tinycore": &webboot.Distro{
+			"boot/vmlinuz64",
```
Comment (bjakobson) [inline webboot/webboot.go]: //kernel from TCs' ISO

**Comment by bjakobson** [comment]
there's a few errors. when I copied the command that failed: 

go run github.com/u-root/u-root/. -files /sbin/iwconfig -files /sbin/iwlist -files /sbin/wpa_supplicant -files /sbin/wpa_action -files /sbin/wpa_cli -files /usr/bin/wpa_passphrase -build=bb all github.com/u-root/webboot/webboot/. github.com/u-root/NiChrome/cmds/wifi

I had to export GOMODULE111=off, then I am out of memory. I'm not sure if either of these is what caused azure to fail...

File: webboot/webboot.go [hunk 2]
```diff
@@ -33,6 +33,11 @@ import (
 	"github.com/u-root/webboot/pkg/webboot"
 )
 
+const tcUrl = "http://tinycorelinux.net/10.x/x86_64/release/TinyCorePure64-10.1.iso"
```
Comment (rminnich) [inline webboot/webboot.go]: if you want:
const (
blah = ""
blee = ""

and so on
)
no need to repeat const. I will approve and leave it up to you.

**Comment by orangecms** [comment]
Correcy @rminnich - no distro ISO works upfront because none of them have pmem support, so we can only boot them with the customized kernels, just like with TinyCore, hence the second commands, following the pattern.

File: webboot/webboot.go (hunk 2 above)
Comment (orangecms) [reply webboot/webboot.go]: Done, looks much nicer indeed. :)

**Pull request merged**